region_code_filter.py

```diff
@@ -370,6 +406,37 @@ def get_landsat_level1_file_paths(
                     fid.write(_fp + "\n")
 
 
+def dict2ard_arg_string(ard_click_params):
+    ard_params = []
+    for key, value in ard_click_params.items():
+        if value is None:
+            continue
+        if key == "test":
+            if value is True:
+                ard_params.append("--" + key)
+            continue
+        ard_params.append("--" + key)
+        # Make path strings absolute
+        if key in ('logdir', 'pkgdir'):
+            value = Path(value).resolve()
+        ard_params.append(str(value))
+    ard_arg_string = " ".join(ard_params)
+    return ard_arg_string
+
+
+def make_ard_pbd(**ard_click_params):
```

Comment:
*make_ard_pbd* or *make_ard_pbs* ?

Reply:
Ha ha.  This is what I like about code reviews.  At least the function is returning a pbs. [I'll fix this!]